Docs update (#4939)
demisto/content: CONTRIBUTING.md

![Content logo](demisto_content_logo.png)

Welcome to Demisto content repo! Contributions are welcome and appreciated.

## How to contribute

> Please read the following guidelines. Following these guidelines will maximize the chances for a fast, easy and effective review process for everyone involved. If something is not clear, please don't hesitate to reach out to us via GitHub, Slack, or email.

Reviewer: I would add the link to difr and the email here instead of in the bottom

* Setup a development environment by following the [Getting Started Guide](docs/getting_started).
* Use the [Package Directory Structure](docs/package_directory_structure) for all Python code. If working on existing code, beyond trivial changes, we require converting to this structure as it allows running linting, unit tests and provides a clearer review process.
* Make sure to read and follow [code conventions](docs/code_conventions).
* Run and verify that the various linters we support pass as detailed [here](docs/linting).

> * For Scripts/Integrations make sure to create unit tests as documented [here](docs/tests/unit-testing)

Reviewer: ```suggestion
* For Scripts/Integrations written in Python, make sure to create unit tests as documented [here](docs/tests/unit-testing)
```

> * Create a test playbook as documented [here](docs/tests). **Note**: for small Scripts that have unit tests, a test playbook is optional.

Reviewer: not sure if `small Scripts` is the way to describe these, maybe "simple" or "straight-forward"

* Validate that our validation hooks pass. If you used `.hooks/bootstrap` as documented in [Getting Started](docs/getting_started) the validation hook will run automatically upon commit. You can also run the validation hooks manually by running `.hooks/pre-commit`.

> * Document you integration a detailed [here](docs/integration_documentation).

Reviewer: ```suggestion
* Document your integration as detailed [here](docs/integration_documentation).
```

* Document your changes in the relevant changelog file as detailed [here](docs/release_notes).
  

>  - Circle CI build must be green [here](https://github.com/demisto/content/tree/master/docs/tests/circleci)

Reviewer: ```suggestion
 - Circle CI build must be green. See further details [here](https://github.com/demisto/content/tree/master/docs/tests/circleci)
```

Author: This was old. I moved this to the review process status checks section.

You should now be ready to create a pull request.

## Create a Pull Request on GitHub

If you are a first time GitHub contributor, please look at these links explaining on how to create a Pull Request to a GitHub repo:
* https://guides.github.com/activities/forking/
* https://help.github.com/articles/creating-a-pull-request-from-a-fork/

**Working on your first Pull Request?** You can learn how from this *free* series [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github)

## Review Process
Once you open a pull request we have a bot (`@content-bot`) that will assist you and provide comments and reminders about the state of the pull request. Additionally, a member of the team will be assigned to review the pull request. Comments will be provided by the team member as the review process progresses.

You will see a few [GitHub Status Checks](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-status-checks) that help validate that your pull request is according to our standards:

> * **ci/circleci: build**: We use [CircleCI](https://circleci.com/gh/demisto/content) to run a full build on each commit of your pull request. The build will run our content validation hooks, linting and unit test. We require that the build pass (green build). If it is failing our content-bot will also add instructive comments about what sections are failing in the build. Follow the `details` link of the status to see the full build UI of CircleCI. More details about our build flow of CircleCI is available [here](docs/tests/circleci).

Reviewer: ```suggestion
* **CI build**: We use [CircleCI](https://circleci.com/gh/demisto/content) to run a full build on each commit of your pull request. The build will run our content validation hooks, linting and unit test. We require that the build pass (green build). If it is failing our content-bot will also add instructive comments about what sections are failing in the build. Follow the `details` link of the status to see the full build UI of CircleCI. More details about our build flow of CircleCI is available [here](docs/tests/circleci).
```

Author: I want to use the same names we show at status checks in Github

Reviewer: 👍
